src/NewButton/styles.ts

```diff
@@ -0,0 +1,233 @@
+import {VariantType} from './types'
+import {Theme} from '../ThemeProvider'
+
+export const TEXT_ROW_HEIGHT = '20px' // custom value off the scale
+
+export const getVariantStyles = (variant: VariantType = 'default', theme?: Theme) => {
+  const style = {
+    default: {
+      color: 'btn.text',
+      backgroundColor: 'btn.bg',
+      boxShadow: `${theme?.shadows.btn.shadow}, ${theme?.shadows.btn.insetShadow}`,
```

Comment:
is `theme` assumed to always be defined inside the fn? If so, do we need the optional chaining? If not, should we provide a fallback?

Reply:
Honestly I only used it to make typescript happy. This seems common usage across our codebase.